src/app/api/organizations/route.ts

```diff
@@ -0,0 +1,15 @@
+import { createOrganization } from '@/server/actions/Organizations';
+import { NextRequest, NextResponse } from 'next/server';
+
+// @route Post /api/organizations/ - Creates an organization
+
+export async function POST(request: NextRequest) {
+  const req = await request.json();
+
+  const res = await createOrganization(req);
+  if (!res) {
+    return NextResponse.json({ message: 'Conflict' }, { status: 409 });
```

Comment:
are we sure that not having `res` will be due to a conflict?

Reply:
Online it said the appropriate HTTP response to a failed POST request should be a conflict. If the POST request fails res should res be null or undefined.

Reply:
I think we should do 500 instead of 409, looking at [this](https://www.restapitutorial.com/httpstatuscodes.html) it looks like conflict is mostly used for PUT requests. Also, 4** responses usually indicate something was wrong from the client, whereas 5** responses indicate something messed up in the server. if there isnt a result from the server function, then it would be due to a server error.